config/percep/config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "frame_write_interval": 10,
+    "buffer_iterations": 20,
+    "DEFAULT_TAG_VAL": -1,
+    "mm_per_m":1000,
+    "threshold_confidence":90,
+
+    "pt_cloud":
+    {   
+        "max_field_of_view_angle": 70,
+        "pt_cloud_width": 320,
+        "pt_cloud_height": 180,
+        "rover_w_mm": 1168,
+        "half_rover": 584,
+        "centerx": 0,
+        "downsample_voxel_filter": 20.0,
+       
+        "ransac": {
+            "max_iterations": 400,
+            "segmentation_epsilon": 10,
+            "distance_threshold": 100
+        },
+
+        "pass_through": {
+            "upperBdZ": 7000.0,
+            "upperBdY": 3000.0, 
+            "lowerBd": 0.0
+        },
+
+        "euclidean_cluster": {
+            "cluster_tolerance": 60,
+            "min_cluster_size": 20, 
+            "max_cluster_size": 10000
+        }
+    },
+
+    "zed_specs":
+    {
+        "resolution_width": 1280,
+        "resolution_height": 720,
+        "focalLength": 2.8
+    },
+
+    "rover_specs":
+    {
+        "zedHeight": 0.4318000078,
+        "realWidth": 1168.400024,
+        "angleOffset": 0.1745329201
+    },
+
+    "alvar_params":
+    {
+        "markerBorderBits": 2,
+        "doCornerRefinement": 0,
+        "polygonalApproxAccuracyRate": 0.08
+
+    }
+} 
```

Comment:
More of a question for me than it is a comment for you, but is there a reason why some constants use snake case and a few use camel case? Thought we were moving towards camel case in terms of style guidelines but is this just for the config file in particular?

Reply:
Good point. We'll clean the style up in PR #576